src/babylon.engine.ts

```diff
@@ -88,9 +88,9 @@
         engine.resetTextureCache();
         scene._removePendingData(texture);
 
-        if (onLoad) {
-            onLoad();
-        }
+        texture.onLoadedCallbacks.forEach(function (callback) {
+            callback();

```

Comment:
Should it be a good place to splice the onLoadedCallbacks array to clean things up?